articles/clients/client-types.md

```diff
@@ -16,6 +16,21 @@ The OAuth 2.0 specification [defines two types of clients](https://tools.ietf.or
 
 When creating a client through the [Dashboard](${manage_url}/#/clients), Auth0 will ask you what type of application you want the client to represent and use that information to determine the client type.
 
+### Checking Your Client Type
+
+You can use the Management API's Get a Client endpoint to check your existing Client's type. If the client is first party, the `is_first_party` equals `true`, else `false`. Be sure to replace `CLIENT_ID` with the ID of your Client.
+
+```har
+{
+	"method": "GET",
+	"url": "https://${account.namespace}.auth0.com/api/v2/clients/CLIENT_ID?fields=is_first_party&include_fields=true",
+	"headers": [{
+		"name": "Authorization",
+		"value": "Bearer MGMT_API_ACCESS_TOKEN"
+	}]
+}
+```
+
```

Comment:
Add a note under this snippet, linking to the doc explaining how to get a mgmt API token